### ISSUE TITLE
Fetching issue with surpas.io

### ISSUE THREAD

**Issue by doc75** [wallabag/wallabag#3747]
### Issue details

When grabbing an article from supras.io it is stripped while https://f43.me/feed/test is grabbing it properly

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.3.3
* How did you install wallabag? Via `git clone`
* Last wallabag version that did not exhibit the issue (if applicable): Unknown
* php version: 7.0.32-0ubuntu0.16.04.1
* OS: Ubuntu 16.04 server
* type of hosting: dedicated
* which storage system you choose at install: MariaDB

### Steps to reproduce/test case

Grab the article: https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/

You will see that the article is grabbed only up to "nos chers framasoftiens".

Here is the content of var/logs/prod.log with Debug mode activated:
```
 
[2018-10-14 21:37:23] request.INFO: Matched route "reload_entry". {"route":"reload_entry","route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::reloadAction","id":"1338","_route":"reload_entry"},"request_uri":"https://web.virlet.com/reload/1338","method":"GET"} []
[2018-10-14 21:37:23] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2018-10-14 21:37:23] security.DEBUG: User was reloaded from a user provider. {"username":"<removed>","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2018-10-14 21:37:23] app.DEBUG: Restricted access config enabled? {"enabled":1} []
[2018-10-14 21:37:23] graby.DEBUG: Graby is ready to fetch [] []
[2018-10-14 21:37:23] graby.DEBUG: . looking for site config for supras.io in primary folder {"host":"supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: Appending site config settings from global.txt [] []
[2018-10-14 21:37:23] graby.DEBUG: . looking for site config for global in primary folder {"host":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: ... found site config global.txt {"host":"global.txt"} []
[2018-10-14 21:37:23] graby.DEBUG: Cached site config with key: supras.io {"key":"supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: . looking for site config for global in primary folder {"host":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: ... found site config global.txt {"host":"global.txt"} []
[2018-10-14 21:37:23] graby.DEBUG: Appending site config settings from global.txt [] []
[2018-10-14 21:37:23] graby.DEBUG: Cached site config with key: global {"key":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: Cached site config with key: supras.io.merged {"key":"supras.io.merged"} []
[2018-10-14 21:37:23] graby.DEBUG: Fetching url: https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/ {"url":"https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying using method "get" on url "https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/" {"method":"get","url":"https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/"} []
[2018-10-14 21:37:23] graby.DEBUG: Use default user-agent "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/535.2 (KHTML, like Gecko) Chrome/15.0.874.92 Safari/535.2" for url "https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/" {"user-agent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/535.2 (KHTML, like Gecko) Chrome/15.0.874.92 Safari/535.2","url":"https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/"} []
[2018-10-14 21:37:23] graby.DEBUG: Use default referer "http://www.google.co.uk/url?sa=t&source=web&cd=1" for url "https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/" {"referer":"http://www.google.co.uk/url?sa=t&source=web&cd=1","url":"https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/"} []
[2018-10-14 21:37:23] graby.DEBUG: Auth: no credentials available for host. {"host":"supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: Auth: no credentials available for host. {"host":"supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: Data fetched: [array] {"data":{"effective_url":"https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/","body":"(only length for debug): 27541","headers":"text/html; charset=UTF-8","all_headers":{"set-cookie":"720planBAK=R3744889740; path=/; expires=Sun, 14-Oct-2018 20:38:12 GMT, 720plan=R1790944297; path=/; expires=Sun, 14-Oct-2018 20:42:05 GMT","date":"Sun, 14 Oct 2018 19:39:15 GMT","content-type":"text/html; charset=UTF-8","transfer-encoding":"chunked","server":"Apache","vary":"Accept-Encoding,Cookie","x-powered-by":"PHP/5.4.45","link":"<https://supras.io/?p=721>; rel=shortlink","x-iplb-instance":"17295"},"status":200}} []
[2018-10-14 21:37:23] graby.DEBUG: Treating as UTF-8 {"encoding":"utf-8"} []
[2018-10-14 21:37:23] graby.DEBUG: Opengraph data: [array] {"ogData":[]} []
[2018-10-14 21:37:23] graby.DEBUG: Looking for site config files to see if single page link exists [] []
[2018-10-14 21:37:23] graby.DEBUG: Returning cached and merged site config for supras.io {"host":"supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: No "single_page_link" config found [] []
[2018-10-14 21:37:23] graby.DEBUG: Attempting to extract content [] []
[2018-10-14 21:37:23] graby.DEBUG: Returning cached and merged site config for supras.io {"host":"supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: . looking for site config for fingerprint.wordpress.com in primary folder {"host":"fingerprint.wordpress.com"} []
[2018-10-14 21:37:23] graby.DEBUG: ... found site config .wordpress.com.txt {"host":".wordpress.com.txt"} []
[2018-10-14 21:37:23] graby.DEBUG: Appending site config settings from global.txt [] []
[2018-10-14 21:37:23] graby.DEBUG: . looking for site config for global in primary folder {"host":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: ... site config for global already loaded in this request {"host":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: Cached site config with key: .wordpress.com {"key":".wordpress.com"} []
[2018-10-14 21:37:23] graby.DEBUG: . looking for site config for global in primary folder {"host":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: ... site config for global already loaded in this request {"host":"global"} []
[2018-10-14 21:37:23] graby.DEBUG: Appending site config settings from global.txt [] []
[2018-10-14 21:37:23] graby.DEBUG: Cached site config with key: fingerprint.wordpress.com.merged {"key":"fingerprint.wordpress.com.merged"} []
[2018-10-14 21:37:23] graby.DEBUG: Appending site config settings from fingerprint.wordpress.com (fingerprint match) {"host":"fingerprint.wordpress.com"} []
[2018-10-14 21:37:23] graby.DEBUG: Cached site config with key: fingerprint.wordpress.com {"key":"fingerprint.wordpress.com"} []
[2018-10-14 21:37:23] graby.DEBUG: Strings replaced: 0 (find_string and/or replace_string) {"count":0} []
[2018-10-14 21:37:23] graby.DEBUG: Attempting to parse HTML with libxml {"parser":"libxml"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //meta[@property="og:title"]/@content for title {"pattern":"//meta[@property=\"og:title\"]/@content"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //meta[@property='og:title']/@content for title {"pattern":"//meta[@property='og:title']/@content"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[contains(@class, 'entry-meta')]//a[@rel='author'] for author {"pattern":"//div[contains(@class, 'entry-meta')]//a[@rel='author']"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //meta[@property="article:published_time"]/@content for date {"pattern":"//meta[@property=\"article:published_time\"]/@content"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[@id='content']//span[contains(@class, 'entry-date')] for date {"pattern":"//div[@id='content']//span[contains(@class, 'entry-date')]"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[contains(@class, 'entry-meta')]//time[@pubdate or @pubDate] for date {"pattern":"//div[contains(@class, 'entry-meta')]//time[@pubdate or @pubDate]"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //html[@lang]/@lang for language {"pattern":"//html[@lang]/@lang"} []
[2018-10-14 21:37:23] graby.DEBUG: Language matched: fr-FR {"language":"fr-FR"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //nav to strip element {"pattern":"//nav"} []
[2018-10-14 21:37:23] graby.DEBUG: Stripping 2 elements (strip) {"length":2} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //header to strip element {"pattern":"//header"} []
[2018-10-14 21:37:23] graby.DEBUG: Stripping 2 elements (strip) {"length":2} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //*[@id='comments' or @id='respond'] to strip element {"pattern":"//*[@id='comments' or @id='respond']"} []
[2018-10-14 21:37:23] graby.DEBUG: Stripping 1 elements (strip) {"length":1} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[contains(@class, 'comments')] to strip element {"pattern":"//div[contains(@class, 'comments')]"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[contains(concat(' ',normalize-space(@class),' '),' navigation ')] to strip element {"pattern":"//div[contains(concat(' ',normalize-space(@class),' '),' navigation ')]"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying sharedaddy to strip element {"string":"sharedaddy"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying wpadvert to strip element {"string":"wpadvert"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying commentlist to strip element {"string":"commentlist"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying sociable to strip element {"string":"sociable"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying related_post to strip element {"string":"related_post"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying wp-socializer to strip element {"string":"wp-socializer"} []
[2018-10-14 21:37:23] graby.DEBUG: Trying addtoany to strip element {"string":"addtoany"} []
[2018-10-14 21:37:23] graby.DEBUG: Stripping 2 empty a elements {"length":2} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[@id="content"]//div[contains(@class, 'entry-content') or contains(@class, 'entrytext') or @class='main' or @class='entry'] for body (content length: 13059) {"pattern":"//div[@id=\"content\"]//div[contains(@class, 'entry-content') or contains(@class, 'entrytext') or @class='main' or @class='entry']","content_length":13059} []
[2018-10-14 21:37:23] graby.DEBUG: Trying //div[@id='content'] for body (content length: 13059) {"pattern":"//div[@id='content']","content_length":13059} []
[2018-10-14 21:37:23] graby.DEBUG: hNews: found hentry [] []
[2018-10-14 21:37:23] graby.DEBUG: hNews: found "1" with .//*[contains(concat(' ',normalize-space(@class),' '),' entry-content ')] [] []
[2018-10-14 21:37:23] graby.DEBUG: Using Readability [] []
[2018-10-14 21:37:23] graby.DEBUG: Detected title: Mise en place d’un relais Tor sous Debian 8 – Supras.io {"title":"Mise en place d’un relais Tor sous Debian 8 – Supras.io"} []
[2018-10-14 21:37:23] graby.DEBUG: Success ? 1 {"is_success":true} []
[2018-10-14 21:37:23] graby.DEBUG: Filtering HTML to remove XSS [] []
[2018-10-14 21:37:23] graby.DEBUG: Returning data (most interesting ones): [array] {"data":{"status":200,"html":"<div id=\"toc_container\" class=\"toc_white no_bullets\">\n<p class=\"toc_title\">Sommaire</p>\n<ul class=\"toc_list\"><li><a href=\"#Mise_en_place\">1 Mise en place</a>\n<ul><li><a href=\"#Preparation_du_terrain\">1.1 Préparation du terrain</a></li>\n<li><a href=\"#Recuperation_des_sources\">1.2 Récupération des sources</a></li>\n<li><a href=\"#Configuration\">1.3 Configuration</a></li>\n<li><a href=\"#Referencement\">1.4 Référencement</a></li>\n</ul></li>\n<li><a href=\"#OEuvrer_pour_son_anonymat\">2 Œuvrer pour son anonymat</a></li>\n</ul></div>\n<p><img class=\"alignnone size-medium wp-image-722\" src=\"https://supras.io/wp-content/uploads/2018/10/Torproject-300x225.png\" alt=\"\" width=\"300\" height=\"225\" srcset=\"https://supras.io/wp-content/uploads/2018/10/Torproject-300x225.png 300w, https://supras.io/wp-content/uploads/2018/10/Torproject-768x576.png 768w, https://supras.io/wp-content/uploads/2018/10/Torproject-1024x768.png 1024w, https://supras.io/wp-content/uploads/2018/10/Torproject.png 1200w\" /></p>\n<p>Il m’arrive par moment d’utiliser le <a href=\"https://www.torproject.org/projects/torbrowser.html.en?\" target=\"_blank\" rel=\"noopener\">Tor Browser</a> et, franchement, le travail fait autour de ce projet d’anonymat et de chiffrement est tout simplement fabuleux. Quoi de plus normal que d’apporter ma (petite) pierre à l’édifice et agrandir ce réseau décentralisé.</p>\n<p>Pour rappel, comme le dit si bien Wikipédia, Tor est un « réseau informatique superposé mondial et décentralisé. » On y retrouve donc ici un moyen de naviguer anonymement sur Internet, à l’heure ou le débat sur la vie privée, le profilage publicitaire et la récolte des données personnelles fait rage. Si vous avez quelques minutes devant vous, je vous invite à lire ce très bon billet de nos chers <a href=\"https://framablog.org/2016/05/06/anonymat-en-ligne-nos-oignons/\" target=\"_blank\" rel=\"noopener\">framasoftiens</a> 🙂</p>\n<p>Tor utilise le principe de routage en oignon :</p>\n<p><img class=\"alignnone wp-image-723\" src=\"https://supras.io/wp-content/uploads/2018/10/routing_onion-300x197.png\" alt=\"\" width=\"439\" height=\"288\" srcset=\"https://supras.io/wp-content/uploads/2018/10/routing_onion-300x197.png 300w, https://supras.io/wp-content/uploads/2018/10/routing_onion-768x505.png 768w, https://supras.io/wp-content/uploads/2018/10/routing_onion.png 875w\" /></p>\n<p>(pour une explication un plus technique, ce qui n’est pas le but de ce billet, c’est à retrouver sur ce très bon <a href=\"http://www.ibibliotech.fr/4641-a-la-decouverte-du-reseau-tor\" target=\"_blank\" rel=\"noopener\">article</a>)</p>\n<p>Lorsque l’on souhaite mettre en place un relais Tor sur son serveur, avec le principe du routage en oignon, plusieurs types de nœuds sont possibles :</p>\n<p>– <strong>Relais de garde (relais A) </strong>: Le relais de garde est le premier nœud qui compose le circuit Tor. C’est un relais qui a commencé au milieu avant de passer garde grâce à un débit stable et rapide (2Mo/s selon le <a href=\"https://trac.torproject.org/projects/tor/wiki/TorRelayGuide\" target=\"_blank\" rel=\"noopener\">guide</a> de mise en place.)</p>\n<p>– <strong>Relais du milieu (relais B)</strong> : Le relais du milieu quand à lui agit comme un deuxième saut, entre le nœud de « garde » et le nœud de sortie.</p>\n<p>– <strong>Relais de sortie (relais C) </strong>: C’est celui par lequel le trafique sort, l’adresse IP publique visible sera donc celle de ce relais. C’est donc ce relais la qui est le plus exposé d’un point de vue <a class=\"c1\" href=\"https://www.torproject.org/eff/tor-legal-faq.html.en\" target=\"_blank\" rel=\"noopener\">juridique</a> alors <strong>prudence</strong>.</p>\n<h1>Mise en place</h1>\n<h2>Préparation du terrain</h2>\n<p>Dans mon cas, des paquets supplémentaires non-présents sur mon système étaient nécessaires pour assurer le bon déroulement de l’installation</p>\n<blockquote>\n<pre>\n$ apt-get update\n$ apt-get install gnupg2 apt-transport-https -y\n</pre></blockquote>\n<h2>Récupération des sources</h2>\n<blockquote>\n<pre>\n$ echo \"deb https://deb.torproject.org/torproject.org jessie main\ndeb-src https://deb.torproject.org/torproject.org jessie main\" &gt;&gt; /etc/apt/sources.list\n</pre></blockquote>\n<p>Ajout des clés GPG</p>\n<blockquote>\n<pre class=\"western\">\n$ gpg2 --recv A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89\n$ gpg2 --export A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89 | apt-key add -\n</pre></blockquote>\n<p>On met à jour la liste des paquets disponibles avec nos nouvelles sources et on passe à l’installation du paquetage Tor</p>\n<blockquote>\n<pre>\n$ apt-get update\n</pre>\n<pre>\n$ apt-get install tor deb.torproject.org-keyring -y\n</pre></blockquote>\n<h2>Configuration</h2>\n<p>Personnalisation des paramètres de mon relais Tor (inspiré de ce <a class=\"c1\" href=\"https://github.com/coldhakca/tor-relay-bootstrap/blob/master/etc/tor/torrc\" target=\"_blank\" rel=\"noopener\">template</a>)</p>\n<blockquote>\n<pre>\n$ vim /etc/tor/torrc\n</pre></blockquote>\n<p>Et on rajoute à la fin :</p>\n<blockquote>\n<pre class=\"western\">\n#Le nom du relais\nNickname MonRelaisDeLaMort\n#Contacter l’hébergeur du relais\nContactInfo exemple@blablabla.org\n#On le lance en tant que daemon\nRunAsDaemon 1\n#Port du relais\nORPort 9001\n#Numéro de port pour le référencement du relais\nDirPort 9030\n#Je n’accepte pas les connexions SOCKS\nSocksPort 0\n#Paramètres important si l’on ne veux pas être définis comme nœud de sortis\nSocksPolicy reject *\nExitpolicy reject *:*\n#Utilisation moyenne de la bande-passante\nRelayBandwidthRate 1200 Kbytes\n#Limite maximale\nRelayBandwidthBurst 1400 KBytes\n</pre></blockquote>\n<p>On redémarre ensuite le service</p>\n<blockquote>\n<pre>\n$ systemctl restart tor\n</pre></blockquote>\n<p>On peux ensuite controller l’état du service</p>\n<blockquote>\n<pre>\n$ systemctl status tor\n</pre></blockquote>\n<p>Un message dans les logs systèmes nous confirme le bon fonctionnement du relais 🙂</p>\n<blockquote>\n<pre>\n$ cat /var/log/syslog |grep Excellent\nOct 6 20:33:05 supras Tor[31112]: Self-testing indicates your ORPort is reachable from the outside. Excellent. Oct 6 20:33:05 supras Tor[31112]: Self-testing indicates your DirPort is reachable from the outside. Excellent. Publishing server descriptor.\n</pre></blockquote>\n<h2>Référencement</h2>\n<p>On peux ensuite vérifier l’état du relais après quelques heures d’activité, lui laissant suffisamment de temps pour se faire référencer : <a class=\"c1\" href=\"https://metrics.torproject.org/rs.html\" target=\"_blank\" rel=\"noopener\">Tor Metrics</a></p>\n<h1>Œuvrer pour son anonymat</h1>\n<p>Si vous souhaitez faire une donation pour soutenir le Tor Project, c’est par ici que cela se passe : <a href=\"https://donate.torproject.org/\" target=\"_blank\" rel=\"noopener\">https://donate.torproject.org/</a></p>\n<p>On a aussi du côté de notre beau pays l’association <a class=\"c1\" href=\"https://nos-oignons.net/\" target=\"_blank\" rel=\"noopener\">Nos Oignons</a> qui maintient des nœuds de sorties, et qui est actuellement en pleine <a href=\"https://nos-oignons.net/campagne2018/\" target=\"_blank\" rel=\"noopener\">campagne de financement</a> (car oui, cela à un coût, malgré toute la bonne volonté et les efforts du monde).</p>","title":"Mise en place d’un relais Tor sous Debian 8 – Supras.io","language":"fr-FR","date":null,"authors":[],"url":"https://supras.io/mise-en-place-dun-relais-tor-sous-debian-8/","content_type":"text/html","open_graph":[],"native_ad":false,"all_headers":{"set-cookie":"720planBAK=R3744889740; path=/; expires=Sun, 14-Oct-2018 20:38:12 GMT, 720plan=R1790944297; path=/; expires=Sun, 14-Oct-2018 20:42:05 GMT","date":"Sun, 14 Oct 2018 19:39:15 GMT","content-type":"text/html; charset=UTF-8","transfer-encoding":"chunked","server":"Apache","vary":"Accept-Encoding,Cookie","x-powered-by":"PHP/5.4.45","link":"<https://supras.io/?p=721>; rel=shortlink","x-iplb-instance":"17295"}}} []
[2018-10-14 21:37:23] app.DEBUG: DownloadImagesSubscriber: disabled. [] []
[2018-10-14 21:37:23] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2018-10-14 21:37:23] request.INFO: Matched route "view". {"route":"view","route_parameters":{"_controller":"Wallabag\\CoreBundle\\Controller\\EntryController::viewAction","id":"1338","_route":"view"},"request_uri":"https://web.virlet.com/view/1338","method":"GET"} []
[2018-10-14 21:37:23] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2018-10-14 21:37:23] security.DEBUG: User was reloaded from a user provider. {"username":"<removed>","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2018-10-14 21:37:23] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
[2018-10-14 21:37:24] request.INFO: Matched route "fos_js_routing_js". {"route":"fos_js_routing_js","route_parameters":{"_controller":"fos_js_routing.controller:indexAction","_format":"js","_route":"fos_js_routing_js"},"request_uri":"https://web.virlet.com/js/routing?callback=fos.Router.setData","method":"GET"} []
[2018-10-14 21:37:24] request.INFO: Matched route "annotations_get_annotations". {"route":"annotations_get_annotations","route_parameters":{"_controller":"Wallabag\\AnnotationBundle\\Controller\\WallabagAnnotationController::getAnnotationsAction","_format":"json","entry":"1338","_route":"annotations_get_annotations"},"request_uri":"https://web.virlet.com/annotations/1338?entry=1338","method":"GET"} []
[2018-10-14 21:37:24] security.DEBUG: Read existing security token from the session. {"key":"_security_secured_area"} []
[2018-10-14 21:37:24] security.DEBUG: User was reloaded from a user provider. {"username":"<removed>","provider":"Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider"} []
[2018-10-14 21:37:24] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []
```

I was not able to find if the stripping elements mentionned in the Debug log are the reason for this partial grab, and I do not know where those stripping information are coming from (I cannot find them in the global.txt file).

I also tried to create a file named supras.io.txt in `vendor/j0k3r/graby-site-config/` directory, but no success:
```
title: //h1[contains(concat(' ',normalize-space(@class),' '),' entry-title ')]
body: //div[contains(concat(' ',normalize-space(@class),' '),' entry-content ')]
```

The article seems to be fetched properly from the log, but some stripping seems to appear.
Is there any way to prevent all stripping or to de-activate some of them ?

**Comment by doc75** [comment]
@techexo I just tried again and still the article is stripped after the second paragraph right at the smiley.
I noticed another article a couple of days ago where the grab stopped at a smiley. Could it be a reason for stopping the grab ?

**Comment by techexo** [comment]
@doc75 What I find very uncanny is that in your debug log the whole body is present! So it is not really a fetching issue, probably more an encoding issue somewhere.

It seems that what is fetched for smileys is not the actual svg picture, but the `alt` property.
Original code:
```html
<span style="color: #0000ff;">framasoftiens</span></a> <img draggable="false" class="emoji" alt="🙂" src="https://s.w.org/images/core/emoji/11/svg/1f642.svg"></p>
```
Fetched code:
```html
<p> [...] framasoftiens</a> 🙂</p>
```

**Comment by doc75** [comment]
On my side here is what I got:
```sql
SELECT * FROM INFORMATION_SCHEMA.SCHEMATA;
```

CATALOG_NAME | SCHEMA_NAME        | DEFAULT_CHARACTER_SET_NAME | DEFAULT_COLLATION_NAME | SQL_PATH
------------|------------------|---------------------|--------------------|---------------
def          | wallabag           | utf8mb4                    | utf8mb4_general_ci     | NULL

Do you mean that you have `utf8mb4_unicode_ci` as collation on your side ?

**Comment by techexo** [comment]
On "DEFAULT_COLLATION_NAME", i have `utf8mb4_general_ci`.
However, `wallabag_entry` table have `utf8mb4_unicode_ci` (and consequently the field `content` of this table, storing the content of the articles).

**Comment by doc75** [comment]
OK, you find the reason for my issue as on my side I have `CHARSET=utf8 COLLATE=utf8_unicode_ci`for the `wallabag_entry` table.
I modified the collation by launching the following command:
```sql
ALTER TABLE wallabag_entry  CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```
And now it is working fine.

But I discovered that other table should have different charset and collation.
My parameter file has `database_charset: utf8` instead of `database_charset: utf8mb4`
Therefore I exported all my articles to JSON format.
Reinstall from scratch Wallabag and reimport the article.

@techexo Thanks a lot for the time taken to help me sorting out this issue, it is really appreciated.
